cps/main.c

```diff
@@ -1272,6 +1275,18 @@ run_cps(struct net_config *net_conf, struct gk_config *gk_conf,
 		goto mailbox;
 	}
 
+	cps_conf->arp_mp = (struct rte_mempool *)rte_mempool_create(
+		"arp_request_pool", (1 << cps_conf->arp_max_entries_exp) - 1,
+		sizeof(struct arp_request), cps_conf->arp_mem_cache_size,
+		0, NULL, NULL, NULL, NULL, socket_id, 0);
```

Comment:
This memory pool is only used by CPS, so we don't need the parameter `cps_conf->arp_mem_cache_size` since it should be zero. Moreover, this memory pool has a single producer and a single consumer, so choose this set of operations.

This comment also applies to patch **cps: have a memory pool for ND requests**.

Reply:
Style: the new memory pools are used in `cps_scan()` and to process the KNI, so it makes sense to create them before those two in the initialization order.